linux_os/guide/services/ssh/sshd_approved_macs.var

```diff
@@ -11,7 +11,7 @@ operator: equals
 interactive: false
 
 options:
-    stig: hmac-sha2-512,hmac-sha2-256
+    stig: hmac-sha2-512,hmac-sha2-256,hmac-sha2-512-etm@openssh.com,hmac-sha2-256-etm@openssh.com
```

Comment:
It is aligned in all products and versions using this stig variable?

Reply:
I have added new option for the extended options as it seems that SLES did not get this update.

Reply:
Thanks @Mab879 . I saw now that one test is failing in `harden_sshd_macs_opensshserver_conf_crypto_policy` rule. Could you update the test scenario, please? Maybe using the variable value instead of a profile would make the test scenario more flexible to future changes.